### DISCUSSION TITLE
[TestOps] Identify launches with unresolved test results.

### DISCUSSION THREAD

**Discussion by quldude** [allure-framework/allure2#1888] [TestOps]
Is there a way to identify which launch has unresolved test results without having to go into every failed launch?

**Comment by cheshi-mantu** [comment]
@quldude hi there.
Not at the moment.
We're thinking about a feature that will allow automatically closing launches which have no unresolved failures.

**Reply by cheshi-mantu** [reply]
@quldude sir, if you need this feature, please describe it in details here: https://help.qameta.io using your corporate email, please
